localnet.md

```diff
@@ -0,0 +1,148 @@
+# V0 LocalNet <!-- omit in toc -->
+
+<!-- https://docs.google.com/presentation/d/1mk0XogopENCI_4WXXvSYm1_DG8EhRLIpwpZQNIA5vqM/edit#slide=id.p -->
+
+- [Prerequisites](#prerequisites)
+  - [Dependencies](#dependencies)
+  - [Download Repos](#download-repos)
+  - [Verify Directory Structure](#verify-directory-structure)
+- [Playground](#playground)
+  - [Setup E2E Stack Environment](#setup-e2e-stack-environment)
+  - [Pocket E2E Stack](#pocket-e2e-stack)
+    - [1. Prepare Stack](#1-prepare-stack)
+    - [2. Run Stack](#2-run-stack)
+    - [3. Watch the network](#3-watch-the-network)
+    - [4. Cleanup Stack](#4-cleanup-stack)
+    - [5. Cleaning containers](#5-cleaning-containers)
+  - [Tx Bot](#tx-bot)
+  - [Configuration changes](#configuration-changes)
+- [TODO](#todo)
+
+## Prerequisites
+
+### Dependencies
+
+This repository is intended to be deprecated within the next 12 months as of writing this document. It is only intended for experienced Pocket developers.
+
+It is intentionally non-exhaustive so newcomers may find it difficult to follow as it is not the intended audience.
+
+### Download Repos
+
+```bash
+mkdir v0_localnet
+cd v0_localnet
+
+git clone git@github.com:pokt-network/pocket-core.git
+git clone git@github.com:pokt-network/tendermint.git
+git clone git@github.com:pokt-network/tx-bot.git
+git clone git@github.com:pokt-network/pocket-e2e-stack.git
```

Comment:
We can add `--recursive` option so that we don't have to run `git submodule` separately.